service/src/test/java/org/mskcc/smile/service/SampleServiceTest.java

```diff
@@ -546,4 +546,32 @@ public void testInvestigatorIdUpdate() throws Exception {
             }
         }
     }
+
+    /**
+     * Tests if sampleClass from SmileSample level is updated
+     * when there is tumorOrNormal update in the SampleMetadata level
+     * @throws Exception
+     */
+    @Test
+    public void testTumorOrNormalUpdate() throws Exception {
+        String requestId = "MOCKREQUEST1_B";
+        String igoId = "MOCKREQUEST1_B_3";
+
+        SmileSample oldSample = sampleService.getResearchSampleByRequestAndIgoId(requestId, igoId);
+        SampleMetadata oldSampleMetadata = oldSample.getLatestSampleMetadata();
+
+        SampleMetadata sampleMetadata = new SampleMetadata();
+        sampleMetadata.setIgoRequestId(requestId);
+        sampleMetadata.setPrimaryId(igoId);
+        sampleMetadata.setCmoPatientId(oldSampleMetadata.getCmoPatientId());
+        sampleMetadata.setTumorOrNormal("tumor");
```

Comment:
can you confirm the casing of "tumor"? i.e., Tumor or tumor

Reply:
You're right, it is 'Tumor'